doc/classes/MultiplayerPeer.xml

```diff
@@ -55,6 +62,11 @@
 		<member name="refuse_new_connections" type="bool" setter="set_refuse_new_connections" getter="is_refusing_new_connections" default="true">
 			If [code]true[/code], this [MultiplayerPeer] refuses new connections.
 		</member>
+		<member name="transfer_channel" type="int" setter="set_transfer_channel" getter="get_transfer_channel" default="0">
+			The channel to use to send packets.
+			Many network API (e.g. ENet and WebRTC) allow the creation of multiple independent channels which behaves, in a way, like separate connections. This means that reliable data will only block delivery of other packets on that channel, and ordering will only be in respect to the channel the packet is being sent on. Using diffent channels to send [b]different and independent[/b] state updates is a common way to optimize network usage and decrease latency in fast-paced games.
```

Comment:
I suggest consolidating the first two lines into the same paragraph, as very short paragraphs tend to waste vertical space:

```suggestion
			The channel to use to send packets. Many network APIs such as ENet and WebRTC allow the creation of multiple independent channels which behaves, in a way, like separate connections. This means that reliable data will only block delivery of other packets on that channel, and ordering will only be in respect to the channel the packet is being sent on. Using differnt channels to send [b]different and independent[/b] state updates is a common way to optimize network usage and decrease latency in fast-paced games.
```